tests/fixtures/scenarios/atmos-functions/stacks/catalog/component-1.yaml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://atmos.tools/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json
+
+components:
+  # This is a mocked component that will only be used to gather output
+  terraform:
+    component-1:
+      metadata:
+        component: myapp
+      vars:
+        location: Los Angeles
+        lang: en
+        format: ''
+        options: '0'
+        units: m
```

Comment:
We should use a different component for this. We are slamming someone providing a nice service, but we don't even need a component that uses any data sources. Let's create an even more basic component fixture. It can have one variable, and one output that contains the variable.